infra/scalingo/partial_backup/partial_backup.sh

```diff
@@ -25,14 +25,24 @@ echo $tunnel_database_url
 
 echo "- Backup pre-data :"
 time pg_dump --no-privileges --no-owner $tunnel_database_url --section=pre-data > "$BACKUP_PATH"/pre_data.sql
+
 echo "- Backup post-data :"
 time pg_dump $tunnel_database_url --no-privileges --no-owner --section=post-data > "$BACKUP_PATH"/post_data.sql
+
 echo "- Backup raw data :"
+recommendation_seq_id=$(psql -qtAX $tunnel_database_url -c 'select last_value from public.recommendation_id_seq;')
+activity_seq_id=$(psql -qtAX $tunnel_database_url -c 'select last_value from public.activity_id_seq;')
 time pg_dump $tunnel_database_url --exclude-table=recommendation --exclude-table=activity -a -F c  > "$BACKUP_PATH"/data.pgdump
+
 echo "- Backup recommendation linked to booking"
-time psql -Atx $tunnel_database_url -c "COPY (SELECT * FROM recommendation r INNER JOIN booking b on r.id = \"recommendationId\") TO stdout;" > /tmp/reco.csv
+time psql -Atx $tunnel_database_url -c "COPY (select * from activity where activity.id < $activity_seq_id \
+  and (table_name='booking' and verb='update') or (table_name='stock' and verb='insert') or (table_name='user' \
+  and verb='update')) TO stdout;" > /tmp/activity.csv
```

Comment:
tu peux aussi écrire : 
`and (table_name in ('booking', 'user') and verb='update') or (table_name='stock' and verb='insert')`